documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Appliance (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Appliance (OVA) file migrations:
+
+* All OVA files are created by VMware vSphere.
+
+[NOTE]
+====
+Migration of of OVA files that were not created by VMware vSphere but are compatible with  vSphere might succeed. However, migration of such files is not supported by {project-short}. {project-short} supports only OVA files created by VMware vSphere.
+====
+
+* The OVA files are in one or more folders under an NFS shared directory that has the following security settings: `(ro)`.
```

Comment:
1. now that it's `ro` (it was `rw` before), I don't think it makes much sense to even mention it - it is common sense that the files should be readable..
```suggestion
* The OVA files are in one or more folders under an NFS shared directory.
```
2. it would probably be expected to store the compressed OVF packages in the nfs-path, e.g., if we export /mnt/arik then:
/mnt/arik/vm1.ova
/mnt/arik/vm2.ova

this is unfortunately not supported in this version (we'll enable this later)
so we better be explicit about the structure that we expect to avoid confusion
the structure that is supported is either in the form of:
/mnt/arik/folder1/vm1.ova
/mnt/arik/folder1/vm2.ova
or
/mnt/arik/folder1/vm1.ova
/mnt/arik/folder2/vm2.ova

and same when the ova is not a compressed package but a folder, like:
/mnt/arik/folder1/vm1/
/mnt/arik/folder1/vm2/
or
/mnt/arik/folder1/vm1/
/mnt/arik/folder2/vm2/

Reply:
Done.